configure.ac

```diff
@@ -1245,7 +1246,7 @@ AC_LINK_IFELSE(
 
 AC_DEFINE([HAVE_SYSTEM], [HAVE_STD__SYSTEM || HAVE_WSYSTEM], [std::system or ::wsystem])
 
-LEVELDB_CPPFLAGS=
+LEVELDB_CPPFLAGS="-DFALLTHROUGH_INTENDED=@<:@@<:@fallthrough@:>@@:>@"
```

Comment:
Shouldn't this be done in `Makefile.leveldb.include`?

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/21430#issuecomment-873435083).